service/dap/server.go

```diff
@@ -847,6 +873,12 @@ func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
 			return
 		}
 		program = debugbinary
+
+		if mode == "test" && args.Cwd == "" {
```

Comment:
I think this belongs below where you handle Cwd, so all logic for it is in one place.

Reply:
Done.